rosidl_cmake/cmake/rosidl_generate_interfaces.cmake

```diff
@@ -33,11 +33,14 @@ include(CMakeParseArguments)
 # :type DEPENDENCIES: list of strings
 # :param SKIP_INSTALL: if set skip installing the interface files
 # :type SKIP_INSTALL: option
+# :param LINT_INTERFACES: if set lint the interface files using
+#   the ``ament_lint`` package
```

Comment:
this does not do a very good job at specifying what linters are run, but it seems like something that will get out-of-date easily so I thought leaving it generic might be better?